src/java/org/apache/cassandra/tcm/Processor.java

```diff
@@ -74,4 +78,20 @@ default ClusterMetadata fetchLogAndWait(Epoch waitFor)
     ClusterMetadata fetchLogAndWait(Epoch waitFor, Retry.Deadline retryPolicy);
 
     LogState reconstruct(Epoch lowEpoch, Epoch highEpoch, Retry.Deadline retryPolicy);
+
+    default List<ClusterMetadata> reconstructFull(Epoch lowEpoch, Epoch highEpoch)
```

Comment:
`LogState reconstruct`  was added by @ifesdjeen for this patch (though he added in a different ticket) but `LogState` is repetitive to work with in our context and we just want the `List<ClusterMetadata>`, so added this here to simplify Accord logic